autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -18,9 +18,10 @@ import SmallFastProgress from '../../../../../SmallFastProgress';
 
 interface SectionSelectProps {
   id: number;
+  onMounted?: VoidFunction;
```

Comment:
Is this ever actually assigned? I don't see any creation of `SectionSelect` that sets it

Reply:
Good point, I ended up removing this prop when I moved the `fixHeight` functionality up from `CourseSelectCard` to `CourseSelectColumn`